tests/unit/conftest.py

```diff
@@ -178,3 +179,41 @@ def mock_airflow_connection_datarobot_gcp_credentials(mocker, dr_gcp_credentials
         ),
     )
     mocker.patch.dict("os.environ", AIRFLOW_CONN_DATAROBOT_GCP_CREDENTIALS_TEST=conn.get_uri())
+
+
+# For AWS Credentials test
+@pytest.fixture
+def dr_aws_credentials_conn_details():
+    return {
+        "aws_access_key_id": "test_aws_access_key_id",
+        "aws_secret_access_key": "test_aws_secret_access_key",
+        "aws_session_token": "test_aws_session_token",
+        "datarobot_connection": "datarobot_default",
+    }
+
+
+@pytest.fixture()
+def mock_datarobot_aws_credentials(mocker, dr_aws_credentials_conn_details):
+    aws_credentials_create_mock = mocker.Mock(
+        credential_id='test-aws-credentials-id',
+        name='datarobot_aws_credentials_test',
+        credential_type='s3',
+        description="Credentials managed by Airflow provider for Datarobot",
+    )
```

Comment:
Why doesn't this contain the actual secret values?

Reply:
its a mock for the output of Credential.create_s3 method which doesn't contain anything except credential_id, name, credential_type and description